packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2652,35 +2672,40 @@ export class ContainerRuntime
 			return;
 		}
 
-		this.setConnectionStateCore(connected, clientId);
+		this.setConnectionStateCore(canSendOps, clientId);
 	}
 
-	private setConnectionStateCore(connected: boolean, clientId?: string): void {
+	/**
+	 * Raises and propagates connected events.
+	 * @param canSendOps - Indicates whether the container can send ops or not (connected and not readonly).
+	 * @remarks - The connection state from container context used here when raising connected events.
```

Comment:
(`@remarks` blocks don't want a preceding `-`. Only `@param` blocks want them.
```suggestion
	 * @remarks The connection state from container context used here when raising connected events.
```